shared/test/test_redis_table.rb

```diff
@@ -95,11 +96,14 @@ def test_redis_tables
 
     # Test getting multiple tables
     table_map = RedisTable.get_tables(@redis, 'shard1', {'table' => 1, 'table2' => 1})
-    assert_equal(
-      {'table' => {'rows' => [row1, row3]},
-       'table2' => {'rows' => [table2_row1]}},
-      table_map
-    )
+    assert_equal(table_map.size, 2)
```

Comment:
Should be `assert_equal 2, table_map.size`, so error messages are correct. Similarly below.